docs-archived/guides/dsm_sobject.md

```diff
@@ -588,51 +589,49 @@ resource "dsm_sobject" "aes_sobject_example_copy" {
     "APPMANAGEABLE",
     "EXPORT"
   ]
-  enabled         = true
-  expiry_date     = "2026-02-02T17:04:05Z"
-  description     = "sobject description copy"
+  enabled     = true
+  expiry_date = "2026-02-02T17:04:05Z"
+  description = "sobject description copy"
   custom_metadata = {
     key1 = "value1"
   }
   rotation_policy = {
-    interval_days = 20
-    effective_at = "20241130T183000Z"
+    interval_days          = 20
+    effective_at           = "20241130T183000Z"
     deactivate_rotated_key = true
-    rotate_copied_keys = "all_external"
+    rotate_copied_keys     = "all_external"
   }
 }
-
 ```
 
 ## Rotate a security object
 
 ```terraform
 resource "dsm_sobject" "aes_sobject_example_rotate" {
-  name = "aes_sobject_example_rotate"
+  name     = "aes_sobject_example_rotate"
   obj_type = "AES"
   group_id = dsm_group.group.id
   key_size = 256
-  key_ops = ["EXPORT", "ENCRYPT", "DECRYPT", "WRAPKEY", "UNWRAPKEY", "DERIVEKEY", "MACGENERATE", "MACVERIFY", "APPMANAGEABLE"]
-  rotate = "DSM"
-  // Name of the above security object
-  rotate_from = "aes_sobject_example"
+  key_ops  = ["EXPORT", "ENCRYPT", "DECRYPT", "WRAPKEY", "UNWRAPKEY", "DERIVEKEY", "MACGENERATE", "MACVERIFY", "APPMANAGEABLE"]
+  rotate   = "DSM"
+  # Name of the above security object
+  rotate_from = dsm_sobject.aes_sobject_example.name
 }
 ```
 
 ## Destroy a security object
 
 ```terraform
-/*
-Destruction can be done while update only. 
-To destroy a security object, `destruct` parameter should be configured.
-And make enabled as false, this is to avoid the differences while updating other resources.
-*/
+
+# Destruction can be done while update only. 
```

Comment:
"...can be done while update" -> "...can be done only on updates to the security object."

Reply:
Modified.